tool/travis.sh

```diff
@@ -32,7 +32,7 @@ while (( "$#" )); do
   test) echo
     echo -e '\033[1mTASK: test\033[22m'
     echo -e 'pub run test --run-skipped'
-    pub run test --run-skipped -j 1 || EXIT_CODE=$?
```

Comment:
Can you run this a couple of times on Travis? A few months ago it was still flaky, but I think we may have upgraded the Dart SDK since then.